TeachingAssistantProfesor/app/src/main/java/malvolyo/teachingassistantprofesor/AddQuizActivity.java

```diff
@@ -0,0 +1,213 @@
+package malvolyo.teachingassistantprofesor;
+
+import android.content.Intent;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.util.TypedValue;
+import android.view.View;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.RadioButton;
+import android.widget.RadioGroup;
+import android.widget.Toast;
+
+import com.google.firebase.database.DataSnapshot;
+import com.google.firebase.database.DatabaseError;
+import com.google.firebase.database.DatabaseReference;
+import com.google.firebase.database.FirebaseDatabase;
+import com.google.firebase.database.ValueEventListener;
+
+public class AddQuizActivity extends AppCompatActivity {
+    String course;
+    String user;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_add_quiz);
+        Intent intent = getIntent();
+        course = intent.getStringExtra(Constants.COURSE);
+        user = intent.getStringExtra(Constants.USERNAME);
+    }
+
+    @Override
+    protected void onActivityResult(int requestCode, int resultCode, Intent intent) {
+        if(requestCode == Constants.INPUT_CODE) {
+            if (resultCode == RESULT_OK) {
+                String ans = intent.getStringExtra(Constants.INPUT);
+                RadioGroup rg = (RadioGroup) findViewById(R.id.quizAnswers);
+                RadioButton rb = new RadioButton(AddQuizActivity.this);
```

Comment:
You can actually keep this in an xml file and just inflate it, instead of manually doing it every time.